test/controllers/EditSetControllerSpec.scala

```diff
@@ -176,6 +176,24 @@ class EditSetControllerSpec extends BaseSpec {
                   "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths. (B)",
```

Comment:
This should be reformatted (`sbt scalafmtAll`).

Reply:
The latest commit isn't formatted.

Reply:
@tna-erasmos please see last commit, it's reformatted

Reply:
Confirmed.